### PR TITLE
libstemmer: unstable-2017-03-02 -> 2.2.0

### PR DESCRIPTION
**Pull request by fpletz**
###### Description of changes

Bump & use new repo.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/libstemmer/default.nix [hunk 1]
```diff
@@ -1,17 +1,30 @@
...
+
+  installPhase = ''
+    runHook preInstall
+    install -Dt $out/lib libstemmer.a
+    install -Dt $out/include include/libstemmer.h
+    runHook postInstall
+  '';
 
   meta = with lib; {
     description = "Snowball Stemming Algorithms";
```
Comment (jtojnar) [inline pkgs/development/libraries/libstemmer/default.nix:30]: New website: https://snowballstem.org/

Comment (fpletz) [reply pkgs/development/libraries/libstemmer/default.nix:30]: Thanks!

**Comment by Artturin** [comment]
broke the cross-compile of libstemmer

```
sed 's/@MODULES_H@/modules.h/' libstemmer/libstemmer_c.in >libstemmer/libstemmer.c
libstemmer/mkmodules.pl libstemmer/modules.h src_c libstemmer/modules.txt libstemmer/mkinc.mak
./snowball algorithms/arabic.sbl -o src_c/stem_UTF_8_arabic -eprefix arabic_UTF_8_ -r ../runtime -u
/nix/store/p7bpdnxqd3i5hwm92mrscf7mvxk66404-bash-5.1-p16/bin/bash: line 4: ./snowball: cannot execute binary file: Exec format error
make: *** [GNUmakefile:251: src_c/stem_UTF_8_arabic.h] Error 126
```

**Comment by Artturin** [comment]
https://github.com/NixOS/nixpkgs/pull/189006

**Pull request merged**